docs/workforce-management-framework/topics/pro-keycloak-enablement.adoc

```diff
@@ -59,36 +151,80 @@ NOTE: The `keycloakConfig` field is partially implemented in link:{WFM-RC-CoreUR
+realm. Ensure that the values for the fields bearer-only, ssl-required and use-resource-role-mappings are not changed.
+For more information, see the {Keycloak} link:{WFM-RC-KeycloakURL}securing_apps/topics/oidc/nodejs-adapter.html[Node.js Adapter documentation].
+
+IMPORTANT: Filling in the realm, auth-server-url and resource fields in the keycloakConfig also enables {Keycloak} in the server application
+
+[discrete]
+=== Keycloak Configuration in the Mobile Application
+The _Keycloak Configuration_ for the mobile application is already filled in. This may be customized by changing the realm, url and clientId fields
+within the application's link:{WFM-RC-AngularJsURL}{WFM-RC-Branch}/demo/mobile/src/config.json[config.json] to values which corresponds to your own client configuration.
+
+NOTE: Ensure that the values for the rest of the fields apart from the realm, url and clientId are not changed.
 
+For more information on Javascript Adapter configurations,
+see the {Keycloak} link:{WFM-RC-KeycloakURL}securing_apps/topics/oidc/javascript-adapter.html[Javascript Adapter] documentation.
 
-[id={context}-enabling-keycloak-on-the-demo-server]
 [discrete]
-== Enabling {Keycloak} on the Demo Server
+=== Keycloak Configuration in the Portal Application
+The _Keycloak Configuration_ for the portal application is already filled in. This may be customized by changing the realm, url and clientId fields
+within the application's link:{WFM-RC-AngularJsURL}{WFM-RC-Branch}/demo/portal/src/config.json[config.json] to values which corresponds to your own client configuration.
+
+NOTE: Ensure that the values for the rest of the fields apart from the realm, url and clientId are not changed.
+
+For more information on Javascript Adapter configurations,
+see the {Keycloak} link:{WFM-RC-KeycloakURL}securing_apps/topics/oidc/javascript-adapter.html[Javascript Adapter] documentation.
+
+[id={context}-enabling-keycloak-on-the-server]
+[discrete]
+== Enabling {Keycloak} on the {PROJECT_NAME} Applications
+
+[discrete]
+=== Enabling {Keycloak} on the Server Application
+. Fill in the Keycloak configuration in the server application.
+. At this point, the routes are protected by {Keycloak}.
+
+See xref:{context}-adding-keycloak-configuration-server[Adding Keycloak Configuration in the Server] for more information.
 
+[discrete]
+=== Enabling {Keycloak} on the Mobile Application
 . Open the file link:{WFM-RC-AngularJsURL}{WFM-RC-Branch}/demo/mobile/src/app/app.js[app.js] in location _/demo/mobile/src/app/_ and change the contents to:
-+
+
 [source,javascript]
 ----
-    // Comment to disable passport auth
-    require('@raincatcher/demo-auth-passport')('wfm-mobile'),
-    // Uncomment to enable Keycloak
-    // require('../keycloak');
+    // Enables passport auth service to be used
+    // require('./passport'),
+    // require('@raincatcher/angularjs-auth')(),
```

Comment:
It will be nice to get that into the ./passport so it will be easy to revert back (people may forget. 
WDYT? new ticket?